src/Orchard.Web/Core/Common/Views/CommonMetadataLastModified.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@*
+    Model:
+      ContentPart
+*@
+
+@using Orchard.ContentManagement
+@using Orchard.Core.Common.Models;
+@{
+    CommonPart commonPart = Model.ContentPart;
+    string lastModifiedBy = commonPart.As<CommonPart>() == null ? null : commonPart.As<CommonPart>().VersionModifiedBy;
+}
+
+@T("By {0}", lastModifiedBy ?? T("unknown").ToString())
```

Comment:
I think that makes sense - just be sure to fall back to "unknown" in case the owner is null or empty as well.